### PR TITLE
fix(latte): parse latte latency and thrpt info correctly

### PR DESCRIPTION
**Pull request by vponomaryov**
None of the `latency`, `mean` and `thrpt` info gets parsed correctly.
Either due to multiple matches or due to multi-line specifics.
Fix the regex's for the latte output to get really proper info.

Also, start searching for the needed info from the summary part instead of the very beginning of output.

### Testing

- [scylla-staging/valerii/vp-rolling-upgrade-latency-regression#16](https://argus.scylladb.com/test/1977e940-c3cd-4776-baac-e2a45c90bce5/runs?additionalRuns[]=0e7f8fb1-b37a-4623-9384-1ff0c77c1dae)

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: sdcm/stress/latte_thread.py [hunk 1]
```diff
@@ -162,21 +162,24 @@ def parse_final_output(result):
         :param result: output of latte stats
         :return: dict
         """
-        ops_regex = re.compile(r'Throughput(.*?)\[op/s\]\s*(?P<op_rate>\d*)\s')
-        latency_99_regex = re.compile(r'\s*99\s*(?P<latency_99th_percentile>\d*\.\d*)\s')
-        latency_mean_regex = re.compile(r'\s*Mean resp. time\s.*\s(?P<latency_mean>\d*\.\d*)\s')
+        ops_regex = re.compile(r'\s*Throughput(.*?)\[op\/s\]\s*(?P<op_rate>\d*)\s')
+        latency_99_regex = re.compile(r'\s* 99 \s*(?P<latency_99th_percentile>\d*\.\d*)\s')
+        latency_mean_regex = re.compile(r'\s*Mean resp. time\s*(?:\[(ms|s)\])?\s*(?P<latency_mean>\d+\.\d+)')
```
Comment (fruch) [inline sdcm/stress/latte_thread.py:167]: maybe we can extend the integration test to cover we parse those right ?

Comment (vponomaryov) [reply sdcm/stress/latte_thread.py:167]: Actually, some existing tests must fail. So, I need to update it.
Also, looks like integration tests are broken after adding of the `LatencyCalculatorMixedResult`.

**Comment by vponomaryov** [comment]
Integration tests will pass after merge and rebase on top of the https://github.com/scylladb/scylla-cluster-tests/pull/8742

**Review by fruch — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]

**Pull request merged**